WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -313,9 +314,19 @@ public void gotXmlRpcEndpoint(String inputSiteAddress, String endpointAddress) {
         slideInFragment(loginUsernamePasswordFragment, true, LoginUsernamePasswordFragment.TAG);
     }
 
+    private void launchHelpshift(String url, String username, HelpshiftHelper.Tag origin) {
+        Intent intent = new Intent(this, HelpActivity.class);
+        // Used to pass data to an eventual support service
+        intent.putExtra(HelpshiftHelper.ENTERED_URL_KEY, url);
+        intent.putExtra(HelpshiftHelper.ENTERED_USERNAME_KEY, username);
+        intent.putExtra(HelpshiftHelper.ORIGIN_KEY, origin);
+        startActivity(intent);
+    }
+
     @Override
-    public void helpWithSiteAddress() {
-        ToastUtils.showToast(this, "Help finding site address is not implemented yet.");
+    public void helpSiteAddress(String url) {
+        launchHelpshift(url, null,
+                HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS, false));
```

Comment:
Maybe it could look something like this:

```
private HelpshiftHelper.Tag getLoginTag(boolean isWPComMode) {
    return HelpshiftHelper.chooseHelpshiftLoginTag(getLoginMode() == LoginMode.JETPACK_STATS, isWPComMode);
}
```